prow/config.yaml

```diff
@@ -405,7 +406,7 @@ presubmits:
       trigger: "(?m)^/test (istio-pilot-e2e-envoyv2-v1alpha3)?,?(\\s+|$)"
       branches:
       - release-0.8
-      - release-1.0.0-snapshot.0
```

Comment:
Let's add release-1.0 instead. It should be release-1.0 instead of release-1.0.0.